packages/react-i18n/src/i18n.ts

```diff
@@ -250,7 +248,12 @@ function isSameDate(source: Date, target: Date) {
   );
 }
 
-function isYesterday(date: Date) {
+export function isToday(date: Date) {
```

Comment:
Can you group all date related functions in a separate `date` file?
I see we have `isToday` `isYesterday`, `isSameMonthAndYear`. 
That would also be a place where we can put more utilities.
I'm thinking about https://github.com/Shopify/javascript-utilities/blob/master/src/dates.ts right now, but we'll add as we might need them.

Reply:
https://github.com/Shopify/javascript-utilities/pull/44

Reply:
Done.